tests/memgraph/test_query_modules.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from gqlalchemy import Memgraph
+from gqlalchemy.exceptions import GQLAlchemyFileNotFoundError
+
```

Comment:
Name of this file might be a bit confusing, there is a `test_query_modules.py` within `tests/graph_algorithms/`, and this tests the addition of query modules, so these tests could either be in `test_memgraph.py`, or be named more specifically, something like `test_add_query_modules.py`